docs/src/plots.md

```diff
@@ -32,13 +33,34 @@ favours the categorical variable for the X axis:
 ```@example Makie
 Makie.rainclouds(A)
 ```
+
+## AlgebraOfGraphics.jl
+
+AlgebraOfGraphics.jl is a high-level plotting library built on top of Makie.jl that provides a declarative "grammar of graphics" for creating complex visualizations. It allows you to construct plots using algebraic operations like `*` and `+`, making it easy to create sophisticated graphics with minimal code.
```

Comment:
I now that the wording is mainly inspired by ggplot (the grammar, part).  However, when reading https://aog.makie.org/dev/philosophy/ we see that here we are actually doing `Algebra` hence the name of the package .  Maybe rephrase that to something like: `similar to the grammar of graphics in ggplot` or something, without claiming that this one here is `grammar of graphics`.

Reply:
something I could help with here? docs build correctly... so...

Reply:
Thanks for the ping, just addressed this and added some more examples.